app/specs/controllers/StudentsShowCtrl.spec.js

```diff
@@ -12,36 +12,39 @@ function() {
   describe('Students Show Ctrl', function() {
     beforeEach(module('paw'));
 
-    var $controller, controller, expectedStudent;
+    var expectedStudent = expectedStudent = {
```

Comment:
What's this? I haven't seen it before...

I'll change it, but if you intentionally wrote this, please explain it to me.